modules/openthread/platform/diag.c

```diff
@@ -8,20 +8,44 @@
 #include <zephyr/drivers/gpio.h>
 
 #include <openthread/error.h>
+#include <openthread/platform/alarm-milli.h>
 #include <openthread/platform/diag.h>
+#include <openthread/platform/radio.h>
 
 #include "platform-zephyr.h"
 #include "zephyr/sys/util.h"
 
+enum {
+	diag_transmit_mode_idle,
+	diag_transmit_mode_packets,
+	diag_transmit_mode_carrier,
+	diag_transmit_mode_modcarrier
+
```

Comment:
nit: Enum values should be all uppercase.

Reply:
fixed